test/WebJobs.Script.Tests.Integration/Host/WebScriptHostManagerTests.cs

```diff
@@ -237,6 +237,8 @@ public void AddRouteDataToRequest_AddsRequestProperty_WhenRouteDataNotNull()
 
             WebScriptHostManager.AddRouteDataToRequest(mockRouteData.Object, request);
 
+            Assert.True(request.Properties.ContainsKey(HttpExtensionConstants.AzureWebJobsHttpRouteDataKey));
```

Comment:
This additional check doesn't seem to add anything or test what you're trying to test - we already know that the value is not null, because we're accessing it below. What you want to test is that you can call AddRouteDataToRequest a second time and that the returned instance is updated, right?